lib/utils.js

```diff
@@ -1,34 +1,25 @@
-'use strict'
-
-const bind = require('./helpers/bind')
-const isBuffer = require('is-buffer')
+import {
```

Comment:
This will completely break npm run test:unit

Should we use require back?

Reply:
AFAIK will we update the tests later on to be able to handle import.

Reply:
got it. i have fixed it in https://github.com/axios/axios/pull/1407